src/zope/dublincore/interfaces.py

```diff
@@ -208,56 +196,52 @@ def setQualifiedRights(qualified_rights):
         The argument must be a sequence of Rights `IDublinCoreElementItem`.
         """
 
+
 class IDCDescriptiveProperties(Interface):
-    """Basic descriptive meta-data properties
-    """
+    """Basic descriptive meta-data properties"""
 
     title = TextLine(
-        title = u'Title',
-        description =
-        u"The first unqualified Dublin Core 'Title' element value."
-        )
+        title=u'Title',
+        description=u"The first unqualified Dublin Core 'Title' element value."
+    )
 
     description = Text(
-        title = u'Description',
-        description =
-        u"The first unqualified Dublin Core 'Description' element value.",
-        )
+        title=u'Description',
+        description=(
+            u"The first unqualified Dublin Core 'Description' element value.")
+    )
+
 
 class IDCTimes(Interface):
-    """Time properties
-    """
+    """Time properties"""
 
     created = Datetime(
-        title = u'Creation Date',
-        description =
-        u"The date and time that an object is created. "
-        u"\nThis is normally set automatically."
-        )
+        title=u'Creation Date',
+        description=u"The date and time that an object is created."
+                    u"\nThis is normally set automatically."
+    )
 
     modified = Datetime(
-        title = u'Modification Date',
-        description =
-        u"The date and time that the object was last modified in a\n"
-        u"meaningful way."
-        )
+        title=u'Modification Date',
+        description=u"The date and time that the object was last modified in a"
+                    u" meaningful way."
+    )
+
 
 class IDCPublishing(Interface):
-    """Publishing properties
-    """
+    """Publishing properties"""
 
     effective = Datetime(
-        title = u'Effective Date',
-        description =
-        u"The date and time that an object should be published. "
-        )
-
+        title=u'Effective Date',
+        description=u"The date and time that an object should be published. "
```

Comment:
If we're cleaning up whitespace inside string literals, why not remove the trailing space here too?

Reply:
Done.